nl-be/auth.json

```diff
@@ -0,0 +1,284 @@
+            "disclaimer": "Door verder te gaan, gaat u akkoord met onze voorwaarden",
+            "disclaimer_terms_of_use_link": "Voorwaarden",
+            "disclaimer_privacy_link": "Privacybeleid"
+        },
+        "secret_page": {
+            "page_title": "Startpagina ${business_name}",
+            "heading": "Welkom bij ${business_name}",
+            "description": "Er is hier nog niets te zien. Maar er komt binnenkort een geweldige startpagina aan.",
+            "sign_in_button": "Inloggen",
+            "manage_business_button": "Account beheren ${business_name}"
+        },
+        "sign_in_confirm_code_page": {
+            "page_title": "Bevestig code | ${business_name}",
+            "heading": "Controleer uw e-mail",
+            "description": "Voer de code in die we zojuist hebben verzonden naar ${email_address}",
+            "warning_too_many_incorrect_attempts": "Er zijn te veel onjuiste codes ingevoerd, dus we hebben een nieuwe verzonden naar ${email_address}",
+            "choose_another_method_link": "Kies een andere methode",
+            "resent_code": "We hebben zojuist een nieuwe code verstuurd naar ${email_address}",
+            "code_input_label": "Code",
+            "continue_button": "Ga verder",
+            "code_input_error_incorrect_code_length": "De bevestigingscode moet uit 6 cijfers bestaan",
+            "code_input_error_resend_requested_too_soon": "Kan nog geen code versturen. Probeer het later nog eens",
+            "code_input_error_incorrect_code": "Voer een geldige bevestigingscode in",
+            "code_input_error_expired_code": "Je bevestigingscode is verlopen, vraag een nieuwe aan",
+            "didnt_receive_a_code_description": "Geen code ontvangen?",
+            "didnt_receive_a_code_link": "Code opnieuw verzenden",
+            "disclaimer": "Door verder te gaan, ga je akkoord met onze voorwaarden",
+            "disclaimer_terms_of_use_link": "Voorwaarden",
+            "disclaimer_privacy_link": "Privacybeleid"
+        },
+        "sign_in_page": {
+            "page_title": "Aanmelden | ${business_name}",
+            "heading": "Welkom terug",
+            "description": "Log in om verder te gaan",
+            "email_input_label": "E-mail",
+            "email_input_error_invalid": "Voer een geldig e-mailadres in",
+            "email_input_error_required": "E-mail is verplicht",
+            "email_input_error_invitation_mismatch": "Het e-mailadres moet overeenkomen met het e-mailadres waarnaar uw uitnodiging is verzonden",
+            "email_input_error_no_account_found": "Geen account gevonden met dit e-mailadres",
+            "auth_form_error_no_auth_methods_enabled": "Sorry, we zien momenteel geen manier om u te verifiëren.",
+            "continue_with_email_button": "Ga verder",
+            "auth_block_separator": "Of",
+            "continue_with_social_provider_button": "Doorgaan met ${social_connection_name]",
```

Comment:
@onderay it seems a few of the properties are ending with a square bracket `]` instead of `}` are you able to fix those up?

Reply:
@onderay we may want to check the spreadsheet as I've seen a few come through as `social_connection_name` when they should be `social_provider_name`.

Same with `enterprise_connection_name` vs `enterprise_provider_name`

Reply:
@DaveOrDead updated